src/Plutus/Certification/Server.hs

```diff
@@ -68,6 +74,8 @@ server ServerCaps {..} eb = NamedAPI
      runConduit
         $ getRuns (setAncestor $ reference ev) rid
        .| evalStateC Queued consumeRuns
+  , abortRun = \rid -> withEvent eb AbortRun \ev -> do
```

Comment:
Based on the type you gave to the `AbortRun` constructor it looks like you intended to add the runId to the event?